src/tests/ffi-key.cpp

```diff
@@ -3231,6 +3231,51 @@ TEST_F(rnp_tests, test_ffi_v6_cert_import)
     rnp_ffi_destroy(ffi);
 }
 
+#if defined(ENABLE_PQC)
+// NOTE: this tests ML-KEM-ipd test vectors
+// The final implementation of the PQC draft implementation will use the final NIST standard.
+TEST_F(rnp_tests, test_ffi_pqc_certs)
+{
+#if !(defined(BOTAN_HAS_ML_KEM_INITIAL_PUBLIC_DRAFT) && defined(ENABLE_PQC_MLKEM_IPD))
+    // we can only verify the test vectors with ML-KEM-ipd
+    GTEST_SKIP();
```

Comment:
Here and in other related tests: It would be better to check whether code safely fails to decrypt data if MLKEM is not available in botan, so no memory leaks/whatever else.